src/adapters/net.cpp

```diff
@@ -306,6 +306,7 @@ namespace net {
    * about the current connection
    */
   string wired_network::linkspeed() const {
+    if (m_linkspeed == -1) return "N/A";
```

Comment:
We could probably just have the function only read:
```cpp
  string wired_network::linkspeed() const {
    return m_linkspeed == -1 ? "N/A" : (to_string(m_linkspeed) + " Mbit/s");
  }
```

Don't really know why 0 should print out `???`

Reply:
Finally remembered to make this change; see https://github.com/polybar/polybar/pull/1772/commits/9809d9e9c75c74f98f863931f949543ff1d3c4aa